helix-lock/src/main/java/org/apache/helix/lock/helix/ZKDistributedNonblockingLock.java

```diff
@@ -54,8 +53,8 @@
    * @param lockMsg the reason for having this lock
```

Comment:
Can you fix the typo "lcok" in this file?

Reply:
Updated. Thanks for checking!